lib/widget/entry_table.rb

```diff
@@ -23,27 +23,62 @@ class Widget::Table::EntryTable < Widget::Table
   detailed_table self
 
   def render
-    content = content_tag :table, { class: "report detail-report", id: "sortable-table" } do
-      concat head
-      concat foot
-      concat body
+    content = content_tag :div, {class: "generic-table--container"} do
+      content_tag :div, {class: "generic-table--results-container"} do
+        table = content_tag :table, {"interactive-table" => true, role:"grid", class: "generic-table", id: "sortable-table" } do
+          concat colgroup
+          concat head
+          concat foot
+          concat body
+        end
+        table + content_tag(:div, class: "generic-table--header-background"){}
```

Comment:
Space missing to the left of {.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.